infra/concourse/Makefile.BUILD

```diff
@@ -14,7 +14,7 @@ BUILD_RUBY_VERSION := 2.6.3
 # Fixing bugs or making trivial changes should be considered a patch release.
 
 DOCKER_TAG_VERSION_TERRAFORM := 2.0.0
-DOCKER_TAG_VERSION_KITCHEN_TERRAFORM := 2.0.0
+DOCKER_TAG_VERSION_KITCHEN_TERRAFORM := 2.0.1
```

Comment:
This should be `2.1.0` as it is a backward compatible feature rather than a patch.

Reply:
Sure, makes sense. Updated!